### PR TITLE
Friendly namespace

### PR DESCRIPTION
**Pull request by tballmsft**
use block attribute on namespace as friendly category name:
```
//% color=#B4009E weight=90
//% block="Shift Instructions"
namespace shift { }
```
Yields

image

### CODE REVIEW

**Comment by pelikhan** [comment]
Could you describe the change?

**Comment by tballmsft** [comment]
It computes into the same variable as before, so whatever we did to localize that holds.

**Comment by pelikhan** [comment]
We get the namespace localization via the blocks loc files.

**Comment by tballmsft** [comment]
So, is there something I need to do? I am just providing a different way for the dev to provide the category name.

**Comment by pelikhan** [comment]
Use the "block" attribute instead and it should just work (TM).

**Comment by tballmsft** [comment]
What is the "block" attribute? There are many "block" attributes.

**Comment by pelikhan** [comment]
There is one called “block” in the metadata. It is currently not used for namespaces.

File: pxtblocks/blocklyloader.ts [hunk 1]
```diff
@@ -176,7 +176,9 @@ namespace pxt.blocks {
             let isAdvanced = nsn && nsn.attributes.advanced;
 
             if (nsn) ns = nsn.attributes.block || ns;
-            let catName = ts.pxtc.blocksCategory(fn);
+            let catName = ns
```
Comment (pelikhan) [inline pxtblocks/blocklyloader.ts]: nitpick: ``let catName = ns || ts.pxtc.blocksCategory(fn)``

**Comment by tballmsft** [comment]
This is failing on pxt-microbit. I think block attribute on namespace is being used to some other effect.

**Comment by tballmsft** [comment]
Working fine for Blockly. Need to add code to do the same for Monaco toolbox: createCategoryElement

**Comment by tballmsft** [comment]
All done.

**Pull request merged**